packaging_automation/.python-version

```diff
@@ -0,0 +1 @@
+3.8.5
```

Comment:
I think you used pyenv to fix the python version. However we are not using pyenv in our CI/CD exec environment. If we use, we need to add a pre execution step to install it which will increase the time needed to finish he job.
I think adding pyenv is not necessary right now

Reply:
I think we do not need to change anything on the CI. It's good practice to store the supported python versions in a python project. The file can stay there for those that wish to setup a virtual environment.

I may as well remove it. Your call.

Reply:
I got it. No need to remove that file.